src/screens/Home/index.tsx

```diff
@@ -1,30 +1,55 @@
+/* eslint-disable react/no-unstable-nested-components */
 import React from 'react';
-import {ListRenderItemInfo, TouchableOpacity} from 'react-native';
+import {ListRenderItemInfo} from 'react-native';
+import {useTheme} from 'styled-components';
 
 import {ArticleJSON} from '~/@types/dtos/everythingDTO';
 
-import {Separator, Text} from '~/components';
+import {Input, Separator} from '~/components';
+import Header from '~/components/Header';
+import Loading from '~/components/Loading';
+import ArticleHeadline from './localComponents/ArticleHeadline';
```

Comment:
```suggestion
import {Input, Separator, Header, Loading } from '~/components';
import ArticleHeadline from './localComponents/ArticleHeadline';
```
We already have an index file to import all components, so, put your need components inside there.